### PR TITLE
feat: upgrade avatarlist logic

### PR DESCRIPTION
**Pull request by Iliustrator**

### CODE REVIEW

File: src/components/AvatarsList/AvatarsList.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+import { auth } from '../../services/firebase';
 import UserAvatar from '../UserAvatar/UserAvatar';
 import classes from './AvatarsList.module.css';
 
 import useHomeApi from '../../hooks/useHomeApi';
 import Spinner from '../Spinner/Spinner';
 
-const AvatarsList = ({ onUserSelected }) => {
+const AvatarsList = ({ onUserSelected, currentUser }) => {
+  const defaultUser = auth.currentUser.uid;
```
Comment (AntonOvc) [inline src/components/AvatarsList/AvatarsList.js:10]: I think `currentUser` is better naming than `defaultUser`

File: src/components/DebtForm/DebtForm.js [hunk 2]
```diff
@@ -50,9 +50,9 @@ const DebtForm = () => {
   return (
     <Fragment>
       <div>
-        <AvatarsList onUserSelected={setSender} />
-        <AvatarsList onUserSelected={setReceiver} />
-      </div>
+        <AvatarsList onUserSelected={setSender} currentUser={receiver}/>
```
Comment (AntonOvc) [inline src/components/DebtForm/DebtForm.js:53]: I think not `currentUser` but `selectedUser`

Comment (AntonOvc) [reply src/components/DebtForm/DebtForm.js:53]: is better for naming

**Review by AntonOvc — CHANGES_REQUESTED** [review]

File: src/components/AvatarsList/AvatarsList.js [hunk 3]
```diff
@@ -27,8 +28,9 @@ const AvatarsList = ({ onUserSelected }) => {
       onSelected={setSelectedId}
       isActive={selectedId === user.id}
       isDisabled={selectedId && selectedId !== user.id}
-      avatarsName={setSelectedName}
+      isBlock={currentUser === user.id || currentUser && currentUser !== defaultUser && user.id !== defaultUser}
```
Comment (AVPcm) [inline src/components/AvatarsList/AvatarsList.js:31]: isBlocked

Comment (AVPcm) [reply src/components/AvatarsList/AvatarsList.js:31]: XXXuser -> XXXuserId (in fact we store only ID's here)

**Review by AntonOvc — APPROVED** [review]

**Pull request merged**